package.json

```diff
@@ -26,7 +26,7 @@
   },
   "scripts": {
     "pretest": "./dbinit.sh ${AUTH_FLAG} init ${MONGODB_VERSION}",
-    "test": "coffeelint index.coffee test/ && mocha --require coffee-script/register --compilers coffee:coffee-script/register",
+    "test": "coffeelint index.coffee test/ && mocha --require coffee-script/register 'test/**/*.coffee'",
```

Comment:
Changed this after updating mocha based on the deprecation of --compilers (see: https://github.com/mochajs/mocha/wiki/compilers-deprecation)